huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_queue_v1_test.go

```diff
@@ -0,0 +1,99 @@
+package dli
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/huaweicloud/golangsdk/openstack/dli/v1/queues"
+	act "github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func TestAccDliQueueV1_basic(t *testing.T) {
+	rName := fmt.Sprintf("tf_acc_test_dli_queue_%s", acctest.RandString(5))
+	resourceName := "huaweicloud_dli_queue_v1.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { act.TestAccPreCheck(t) },
+		Providers:    act.TestAccProviders,
+		CheckDestroy: testAccCheckDliQueueV1Destroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDliQueueV1_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDliQueueV1Exists(resourceName),
```

Comment:
Unit test verification is not sufficient, suggest to add some attributes verification.